src/importexport/guitarpro/internal/guitarbendimport/benddatacollector.cpp

```diff
@@ -256,10 +256,10 @@ void BendDataCollector::moveSegmentsToTiedNotes(tied_chords_bend_data_chunk_t& d
     const size_t segmentsSize = dataForFirstNote.segments.size();
     size_t newSegmentsSize = segmentsSize;
     const Note* note = dataForFirstNote.note;
-    int diff = dataChunk.size() - dataForFirstNote.segments.size();
+    size_t diff = dataChunk.size() - dataForFirstNote.segments.size();
```

Comment:
Is there any guarantee that `dataChunk.size() >= dataForFirstNote.segments.size()`? Otherwise, it looks like we'd get underflow, resulting in an seemingly-infinite loop below.

Also, looks like we could just use `segmentsSize` instead of `dataForFirstNote.segments.size()`)

Reply:
Oh, I was wrong. Here it's actually correct fix by @Jojo-Schmitz, because I check previously before calling that method, that it's bigger. Maybe I should open my PR to covert those fixes?

Reply:
okay, if you wanna fix that (thx a lot), then maybe IF_ASSERT (dataChunk.size() >= dataForFirstNote.segments.size()) would be even better than casting to int

Reply:
```c++
    IF_ASSERT_FAILED(dataChunk.size() >= dataForFirstNote.segments.size()) {
        return;
    }
```

(because the body of the for loop wouldn't be hit anyway, when the int is negative)